about.html

```diff
@@ -0,0 +1,120 @@
+                <div class="mobile-menu">
+                  <div class="close-menu"><button class="menu-button"><img src="./images/close.png" alt="close-button" ></button></div>
+                  <ul class="mobnav">
+                      <li class="mobnav1" onclick="showhide()"><a class="moblink" href="./index.html">home</a></li><hr>
+                      <li class="mobnav1" onclick="showhide()"><a class="moblink" href="#">program</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">Join</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">Sponser</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">News</a></li><hr>
+                    </ul>
+              </div>
+                <div class="desk-links">
+                    <ul class="deskul">
+                        <li class="desklist"><a href="#" class="desklink"><i class="fa-brands fa-facebook-f"></i></a></li>
+                        <li class="desklist"><a href="#" class="desklink"><i class="fa-brands fa-twitter"></i></a></li>
+                        <li class="desklist"><a href="#" class="desklink">English</a></li>
+                        <li class="desklist"><a href="#" class="desklink">My page</a></li>
+                        <li class="desklist"><a href="#" class="desklink">Logout</a></li>
+                    </ul>
+                </div>
+                <div class="desk-nav">
+                    <div class="logolink">
+                      <a class="logolink1" href="#"><span class="desklogo">VR Odyssey</span></a>
+                    </div>
+                    <ul class="deskul2">
+                      <li class="desklist2"><a href="#" class="desklink1">About</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">program</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Join</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Sponser</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">News</a></li>
+                    </ul>
+                    <div class=desbutton><button class="deskbutton">VR Campaign</button></div>
+                </div>
+                <div class="home">
+                    <ul class="homeul">
+                      <li class="homeli"><a href="./index.html" class="desklink2"><i class="fa-solid fa-house homeicon"></a></i></li>
+                      <li class="homeli"><a href="#" class="desklink2"><i class="fa-solid fa-angle-right"></a></i></li>
+                      <li class="homeli"><a href="./about.html" class="desklink2">about</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <section class="page">
+            <p class="para">"Welcome to VR!"</p>
```

Comment:
- [ ] Kindly 😃 try to remove this text from the about page as you see here in the about page in the Behance this text is not found in it, so let's make 👌it like the Behance design by removing the text 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225890380-9c14b18b-e435-4885-a99c-a246343ff7d3.png)    |    ![image](https://user-images.githubusercontent.com/83360636/225890343-35c574a6-493e-4193-ba1d-592129f4a3ee.png)  |